local-antora-playbook.yml

```diff
@@ -7,13 +7,13 @@ content:
   - url: .
     branches: HEAD
   - url: https://github.com/redpanda-data/documentation
-    branches: [v/*, site-search, main, shared]
+    branches: [v/*, site-search, cloud-api-docs-combined, shared]
```

Comment:
Change this back to `main` before merging. This is just here so we can test including content from that branch.

Reply:
^^ @kbatuigas  Reminder - as I see this still reads `main`